src/ts/uiconfig.ts

```diff
@@ -105,4 +105,9 @@ export interface UIConfig {
    * If set to true, prevents the UI from using `localStorage`.
    */
   disableStorageApi?: boolean;
+
+  /**
+   * If set to true, allows seeking inside timeline marker duration. Default is false
+   */
+  seekbarAllowSeekInMarkerDuration?: boolean;
```

Comment:
Given that there is already a config option `seekbarSnappingRange` 

https://github.com/bitmovin/bitmovin-player-ui/blob/bffd82b8951734a3b2f4eb905c42481646f1c1cb/src/ts/uiconfig.ts#L79-L82

 I would propose a name like `seekbarSnappingEnabled` whose default value is `true` (and reverse the logic). This way it's more obvious that the two config options are related. Maybe, also worth clarifying in the doc of the property of `seekbarSnappingRange` that it has no effect if snapping is disabled.

I can take over the PR and do these changes if you agree @luckygoyal-bitmovin 🙂

Reply:
Thanks @jmsn . It will be great if you can takeover the PR.

Reply:
on it 👍